tests/dialects/test_clickhouse.py

```diff
@@ -880,3 +880,18 @@ def test_drop_on_cluster(self):
         for creatable in ("DATABASE", "TABLE", "VIEW", "DICTIONARY", "FUNCTION"):
             with self.subTest(f"Test DROP {creatable} ON CLUSTER"):
                 self.validate_identity(f"DROP {creatable} test ON CLUSTER test_cluster")
+
+    def test_datetime_funcs(self):
+        # Function aliases are roundtripped back to one name e.g. DATE_SUB(), DATESUB() -> DATE_SUB()
+        func_names = (("DATE_SUB", "DATESUB"), ("DATE_ADD", "DATEADD"))
+        for funcs in func_names:
+            # Functions of type <func>(date, unit)
+            for func in (*funcs, ("TIMESTAMP_ADD", "TIMESTAMPADD")):
```

Comment:
This is a bit awkward, not sure if it's written correctly: `*funcs` expands to each element in the tuples (so you get back `DATE_SUB`, DATESUB` and then `("TIMESTAMP_ADD", "TIMESTAMPADD")` in the same iteration.

Can we clean these tests up a bit? They're hard to follow

Reply:
Fixed it and made it more readable, let me know if it's better